src/js/feedback/components/DefaultView.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function GiveUsFeedback({ feedbackButtonClicked }) {
+  return (
+    <div className="usa-grid-full">
+      <div className="usa-width-one-half">
+        <h4 className="feedback-widget-title">Tell us what you think</h4>
+        <p className="feedback-widget-intro">
+          We are always looking for ways to make Vets.gov better.
+        </p>
+      </div>
+      <div className="usa-width-one-half">
+        <button onClick={feedbackButtonClicked} className="usa-button-secondary feedback-button">Give us feedback</button>
```

Comment:
I don't think it hurts to name this `revealForm` instead of `feedbackButtonClicked`. That's what this is going to do, and it's not generic enough that we could do a variety of things based on this button click. So `revealForm` tells you more about what's going to happen when this button is clicked.